Remotion/Validation/Globalization/LocalizedValidationMessageFactory.cs

```diff
@@ -125,7 +125,7 @@ public enum ResourceIdentifier
       if (validator is LessThanValidator)
         return ResourceIdentifier.ValueMustBeLessThanValidationMessage;
 
-      if (validator is LengthValidator lengthValidator)
+      if (validator is ILengthValidator lengthValidator)
```

Comment:
THis needed to be changed to work similiar to the LessThanValidator etc tests in line 125 and previous. The test based on Min/Max Length was only done before we had the LengthValidator as a base type.